pkg/cleantemp/cleantemp.go

```diff
@@ -85,3 +102,45 @@ func CleanTempDir(ctx logContext.Context, dirName string, pid int) error {
 	}
 	return nil
 }
+
+func CleanTempFile(ctx logContext.Context, fileName string, pid int) error {
```

Comment:
Does this method differ from `CleanTempDir` in any way other than inverting the `file.IsDir()` check? (Other than its log messages.) If not, and it's always called alongside `CleanTempFile`, then can they just be collapsed into one function that deletes things irrespective of dir-ness?

Put another way: Instead of adding a new function to delete files and always calling it alongside the old function that deletes directories, could you modify the old function to delete files and directories together? That would result in less code to maintain without creating any new coupling.